python/libraries/bimvee/importNumpy.py

```diff
@@ -0,0 +1,96 @@
+import numpy as np
```

Comment:
Add the standard license text at the beginning of the file, put yourself as author and explain that the format is very specific to your particular experimental set up.